gulpfile.js

```diff
@@ -81,10 +82,15 @@ function buildTask(name, stageList) {
   });
 };
 
+function buildExperimentTask(name, experimentFile) {
+  experimentFiles.push(experimentFile);
+  buildTask(name, [{name: 'input', options: {data: experimentFile}}, 'fileToBuffer', 'bufferToString', 'doExperiment']);
+};
+
 /*
  * Some example pipelines.
  */
-buildTask('html', [{name: 'input', options: {data: options.file}}, 'fileToBuffer', 'bufferToString', 'jsonParse', 'HTMLWriter', {name: 'writeStringFile', options: {filename: 'result.html.html'}}]);
+buildExperimentTask('html', 'tasks/html.exp');
```

Comment:
* We'll keep all the toplevel experiments in a single file, with each digraph named appropriately.
* I'm not sure we want .exp as the format extension - maybe .erlnmyr, or .emr
* these experiments should all be loaded automatically from the file (and when we have extra libraries, they'll be able to contribute their own toplevel experiment file that will also be loaded automatically).